Documentation/DataSources/PrivateWorkbooksDeprecation.md

```diff
@@ -0,0 +1,12 @@
+# Private Workbooks Deprecation
+
+Private Workbooks is going away by the end of 2020. You will still be able to access your private workbooks but any edit or save can only be saved as a Shared Workbook. 
```

Comment:
we'll need to verify the exact timing/wording here.  not sure if it will be by the end of 2020 given deployment freezes around all the holidays.